rune/libenclave/attestation/sgx/attester/attester-ecdsa.go

```diff
@@ -3,15 +3,14 @@ package sgx_attester // import "github.com/inclavare-containers/rune/libenclave/
 import (
 	"fmt"
 	"github.com/inclavare-containers/rune/libenclave/attestation/registration"
-	"github.com/inclavare-containers/rune/libenclave/attestation/sgx"
```

Comment:
Why not keep it in attestation package?

Reply:
> Why not keep it in attestation package?

If we want to keep `attestation` sub-packages internal, we must keep the `attestation/sgx` unimportable. so we need to move constants of the `attestation/sgx` package into the `intelsgx` package

Reply:
Is there any other packages requiring ias package? If not, move it to internal as well.

Reply:
> Is there any other packages requiring ias package? If not, move it to internal as well.

OK, but I also need to move PCCS internal.